utils/sampler/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log for `sampler`
 
+## 0.3.0
+
+* Run the globally activated executable out of a temp dir instead of the 
+  pub cache
+* Remove most of the platform directories (leave macOS entitlements), and 
+  run `flutter create` on the platform to create them.
```

Comment:
Do you have tests in place that detect when changes to the template (and/or the flutter create behavior) break the app?

(That has been a source of undetected breakages for things in flutter/test where a similar approach was used)

Reply:
No, because I can't think of a good way to test it. I mean, I could try just running a test app in a temp dir with the same entitlements, I guess.